uberfire-test-utils/src/main/java/org/guvnor/test/CDITestSetup.java

```diff
@@ -47,7 +50,12 @@ public void setUp() throws Exception {
 
     public void cleanup() {
         if (weld != null) {
-            weld.shutdown();
+            try {
+                weld.shutdown();
+            } catch (NullPointerException npeException) {
+                Logger.getLogger(CDITestSetup.class.getName())
```

Comment:
Just out of curiosity: is there a reason to use java util logging (as opposed to slf4j used in most other places)?

Reply:
I would say no, will update. Thanks.